automation/capitanodoc/utils.ts

```diff
@@ -51,16 +51,6 @@ export function parseCapitanoOption(option: OptionDefinition): string {
 	return ent.encode(result);
 }
 
-/** Convert e.g. 'env add NAME [VALUE]' to 'env add <name> [value]' */
-export function capitanoizeOclifUsage(
-	oclifUsage: string | string[] | undefined,
-): string {
-	return (oclifUsage || '')
-		.toString()
-		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, match => `<${match}>`)
-		.toLowerCase();
-}
```

Comment:
The `capitanoizeOclifUsage()` function was moved from `automation/capitanodoc/utils.ts` to `lib/utils/oclif-utils.ts` so that it could be reused in `lib/actions/help_ts.ts`.  The `automation` folder is not published to the `npm` registry, so the CLI code cannot use it at runtime. (It is intended for use at build time, e.g. to create the markdown documentation for the website.)